bin/install-qa-check.d/60pkgconfig

```diff
@@ -1,12 +1,70 @@
 # Check for pkg-config file issues
 
 pkgconfig_check() {
+	# Make a list of .pc files and bail out if there aren't any
+	mapfile -d '' files < <(find "${ED}"/usr/lib*/pkgconfig "${ED}"/usr/share/pkgconfig -maxdepth 1 -type f -name '*.pc' -print0 2>/dev/null)
+	[[ -n "${files[@]}" ]] || return
+
+	local f
+
 	# Look for leaking LDFLAGS into pkg-config files
-	local f=$(egrep -sH '^Libs.*-Wl,(-O[012]|--hash-style)' "${ED}"/usr/*/pkgconfig/*.pc)
+	f=$(egrep -sH '^Libs.*-Wl,(-O[012]|--hash-style)' "${files[@]}")
 	if [[ -n ${f} ]] ; then
 		eqawarn "QA Notice: pkg-config files with wrong LDFLAGS detected:"
 		eqawarn "${f//${D}}"
 	fi
+
+	# Validate using pkgconfig
+	# Note: some less common implementations may not support this?
+	# Seems like f.d.o, OpenBSD, and of course pkgconf do though.
+	if ! pkg-config --validate "${files[@]}" ; then
+		eqawarn "QA Notice: pkg-config files which fail validation found!"
+		eqawarn "Run 'pkg-config --validate ...' for more information"
+	fi
+
+	# Check for unexpected paths
+	# e.g. https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=c90ab38e3577aae61fac2341b34ad593948de1cd
+	if [[ -n "${EPREFIX}" ]] ; then
```

Comment:
```suggestion
	if [[ -n ${EPREFIX} ]] ; then
```

Quoting inside `[[ ... ]]` is only needed for RHS of `==` (to disable wildcards).

Reply:
Ah, thanks.